src/main/webapp/WEB-INF/views/after.jsp

```diff
@@ -0,0 +1,10 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Taxi Service</title>
+</head>
+<body>
+<h2>Your input is being processed...</h2>
+<p><a href="${pageContext.request.contextPath}/index">Back to main page</a></p>
+</body>
+</html>
```

Comment:
add empty line in the end of the file, the same for other jsps

Reply:
✅